kimchi/src/circuits/expr.rs

```diff
@@ -197,6 +198,23 @@ impl Column {
             Column::Coefficient(i) => format!("c_{{{}}}", i),
         }
     }
+
+    fn text(&self) -> String {
```

Comment:
Practice will tell us if we need a more verbose version of these texts or if single letters is meaningful enough :)

Reply:
Yeah, I just tried to keep it similar to the other versions of printing, which are language specific.  There's a tradeoff of compactness vs verbosity regarding which is clearer for debugging.